package.json

```diff
@@ -9,11 +9,11 @@
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^27.4.0",
-    "@types/node": "^16.11.19",
     "@types/react": "^17.0.38",
     "@types/react-dom": "^17.0.11",
-    "axios": "^0.26.1",
+    "axios": "^0.27.2",
```

Comment:
do we need axios? i see you removed it from miroslava's code

Reply:
yes, d3 fetch the data internally in the useCountry hook, but we still need it to get the user scores and the geolocation database in the Game component